### ISSUE TITLE
Translation osTicket

### ISSUE THREAD

**Issue by antiuser** [osTicket/osTicket#4940]
Hi there! Im working on translation osTicket to Russian Language. And i found few bugs in code.
1) In ticket list - page N of about N ("of about" is not translate translatable)
2) "These fields are requested for new tickets via the %s form" (in include/staff/dynamic-form.inc.php is not translatable)
3) Bug found and corrected (include/staff/templates/dynamic-field-config.tmpl.php)
was:
```
    <div class="span4">
        <div style="margin-bottom:5px"><strong>Editable</strong>
        <i class="help-tip icon-question-sign"
            data-content="<?php echo __('Fields marked editable allow agents and endusers to update the content of this field after the form entry has been created.'); ?>"
            data-title="<?php echo __('Editable'); ?>"></i>
        </div>
    </div>
```
become:
```
    <div class="span4">
        <div style="margin-bottom:5px"><strong><?php echo __('Editable'); ?></strong>
        <i class="help-tip icon-question-sign"
            data-title="<?php echo __('Editable'); ?>"
            data-content="<?php echo __('Fields marked editable allow agents and endusers to update the content of this field after the form entry has been created.'); ?>"></i>
        </div>
    </div>

```
4) In include/staff/ticket-view.inc.php line $recipients = __(' Manage Collaborators'); contain space, and i shoud translate with space too
5) In settings i cannot find how to translate Stora Attachments "In the database", and ACL -> Apply To
6) On dashboard not translatable "Range", Calendar, and ticket statuses on ticket graphic.

Can you help to resolve this issues?
Thank you
osTicket v1.12 (a076918)

**Comment by JediKev** [comment]
@antiuser

Thank you for the post. Unfortunately the software was never labeled as fully translated. There are many strings/phrases that are still not translatable. We are always working to make it fully translatable.

With this being said non-translatable strings are not considered bugs, they are just non-translatable strings.

Cheers.

**Comment by antiuser** [comment]
> @antiuser
> 
> Thank you for the post. Unfortunately the software was never labeled as fully translated. There are many strings/phrases that are still not translatable. We are always working to make it fully translatable.
> 
> With this being said non-translatable strings are not considered bugs, they are just non-translatable strings.
> 
> Cheers.

Yep, im understand it, i can commit new messages.mo.php with new translate phrases (you can diff this with original message.mo.php), but some variables not translatable :(

**Comment by JediKev** [comment]
@antiuser

That’s why I mentioned the project was never labeled as 100% fully translated. If you would like to add some please make a pull and we will review.

Cheers.
